pac4j-couch/pom.xml

```diff
@@ -26,6 +26,11 @@
 			<artifactId>org.ektorp</artifactId>
 			<version>${ektorp.version}</version>
 		</dependency>
+        <dependency>
```

Comment:
Actually, I think it is 4 by default for every file!

If it's not what happened, then maybe you need to install a plugin for your IDE that takes editorconfig into account...